Release 4.16.4 (#259)
blackbaud/skyux-forms: CHANGELOG.md

# 4.16.4 (2021-04-16)

> - Updated the radio group component to be marked as touched after an input field loses focus. [#257](https://github.com/blackbaud/skyux-forms/pull/257)

Reviewer: ```suggestion
- Updated the radio group component to mark it as touched after its radio buttons lose focus. [#257](https://github.com/blackbaud/skyux-forms/pull/257)
```
I think I approved this one yesterday and can't tell what changed, but after looking at it again it struck me that "an input field" seems imprecise because we don't usually refer to radio buttons as input fields. If my change to address this is inaccurate, just let me know. The other tweaks are just to use active voice.
